terraform/bin/manage-stack.sh

```diff
@@ -144,6 +151,107 @@ function s3_bucket_exists {
     return 1
 }
 
+# =============================================================================
+#  Utility functions
+# =============================================================================
+
+function join_by { 
+    local separator="$1" joined="" index=1 e=""
+    shift
+    for e in $*; do
+        if [ "${index}" -gt 1 ]; then
+            joined="${joined}${separator}"
+        fi
+        joined="${joined}${e}"
+        ((++index))
+    done
+    echo -e "${joined}"
+}
+
+# =============================================================================
+#  Configuration functions
+# =============================================================================
+
+PROPERTIES_FS="."
+
+#-------------------------------------------------------------------------------
+# Converts yaml to a properties file, so configuration information can be read 
+# by using properties (simpler with bash only, see get_property)
+#-------------------------------------------------------------------------------
+function parse_yaml {
+    local file_name=$1
+    local prefix=$2
+    if  [ ! -f "${file_name}" ]; then
+       echo "File '${file_name}' does not exist" >&2
+       exit 1
+    fi
+    debug "config-data(file):\n$(cat $file_name)\n"
+    cat $file_name | parse_yaml_from_string
+}
+
+function parse_yaml_from_string {
```

Comment:
Assume you borrowed this from [here](http://stackoverflow.com/questions/5014632/how-can-i-parse-a-yaml-file-from-a-linux-shell-script)? Looks good.